Sources/TuistLoader/Utils/ConfigLocator.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import TSCBasic
+import TuistCore
+import TuistSupport
+
+public protocol ConfigLocating {
+    ///  It traverses up the directory hierarchy until it finds a `Config.swift` file inside `tuist` dictionary
+    /// - Parameter path: Path from where to do the lookup.
+    func locate(at path: AbsolutePath) -> AbsolutePath?
+}
+
+public final class ConfigLocator: ConfigLocating {
```

Comment:
Im not sure it should be here. Please correct me if I placed this file it in a wrong place.

Reply:
The directory location is fine. I was even wondering that it might make sense to delete this class and just add `locateConfig` to `ManifestFilesLocator` since `Config` is a `Manifest` 🤔